src/migrations/dapps/1718910535585_create-trades-and-trade-assets-table.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const TRADES_TABLE = 'trades'
+export const TRADE_ASSETS_TABLE = 'trade_assets'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createExtension('uuid-ossp', { ifNotExists: true })
```

Comment:
This command might fail when executing the migration due to the database role not being an administrator. We might want to leave it out of the execution and do it manually.